recipe/build.sh

```diff
@@ -7,14 +7,20 @@
 mkdir build
 cd build
 
+if [ "$(uname)" == "Darwin" ]; then
+  export CXX=clang++
+  export CC=clang
+else
+  export CXX=g++
+  export CC=gcc
+fi
```

Comment:
Instead of adding these manually, add `toolchain` to the `build` requirement and it will set these for you.
Also replace `libgcc` with `libgfortran` in run requirements.